src/index.js

```diff
@@ -48,6 +50,7 @@ class Dropzone extends React.Component {
       document.addEventListener('dragover', Dropzone.onDocumentDragOver, false)
       document.addEventListener('drop', this.onDocumentDrop, false)
     }
+    this.fileInputEl.addEventListener('click', this.onInputElementClick, false)
```

Comment:
But I still don't get why you have chosen to use refs and DOM API instead of React's one.

Something like this didn't work?

```jsx
{
  ...inputProps,
  onClick: (evt) => { 
    evt.stopPropagation();
    if (typeof inputProps.onClick === 'function') {
      inputProps.onClick()
    }
  }
}
```